routes/routes_impl.py

```diff
@@ -589,6 +592,8 @@ def topo_impl(projection: str, zoom: int, x: int, y: int, shaded_relief: bool):
     """
         Generates topographical tiles
     """
+    if zoom > 7:
+        return send_file("/opt/tiles/blank.png")
```

Comment:
Is this file on dory?

Reply:
No, this was a mistake. This file is included in the shapes tar file on the installation script. I'll change the string to include the global variable `shape_file_dir`